### PR TITLE
@craigspaeth: Rough in some initial content

### PR DESCRIPTION
**Pull request by dzucconi**
This gets something up and running to look at.

I was looking at [this article](https://medium.com/@dhg/parallax-done-right-82ced812e61c) (which I see you [already read](http://static.damonzucconi.com/_capture/aw/d1qynsm1c80ow.png)) and [found a pretty straightforward implementation of the main ideas](https://github.com/strangelove/kubrick).

For touch devices at the moment it just presents the frames as a normal scrollable page. I'll be switching this up to something swipe based once I talk to design about it more.

### CODE REVIEW

File: src/assets/javascripts/timeline/introduction.coffee [hunk 1]
```diff
@@ -0,0 +1,55 @@
+module.exports = [
```
Comment (craigspaeth) [inline src/assets/javascripts/timeline/introduction.coffee]: Very nice!

**Comment by craigspaeth** [comment]
Kubrick is pretty sweet. Very exciting progress!

**Comment by gib** [comment]
very nice!!

**Pull request merged**